front/lib/api/assistant/conversation/helper.ts

```diff
@@ -0,0 +1,41 @@
+import {
```

Comment:
Hesitated with a wrapper here; it's used in ~all api routes with conversations, I ~think the direct function call is more readable although a bit less verbose, open to discussion